### PR TITLE
feat: added config map to populate configuration localhost ip

### LINKED ISSUE #92: [FEAT]: Dynamically populate dev cube localhost ip configuration based on OS version (host.docker.internal vs. actual IP)
Our dev Kube deployment is currently failing in linux based system.  The problem right is not being able to use `host.docker.internal`) on a linux and/or wsl2 linux host.  This is because we can't seem to use this docker directive in our kube configs.

`    extra_hosts:
      - "host.docker.internal:host-gateway"`

### PR DESCRIPTION
**Pull request by tangopapatime**
# Description

Added a config map to the kube configuration so that we can dynamically populate the `localhost` map when running `deploy-dev.sh`.  The config map depends on an `environment-variables.env` file that is updated when running `./deploy-dev.sh`.  This should fix the `host.docker.internal` issue not working on linux/wsl2 based systems.  In the future we can utilize this `environment-variables.env` file for additional key/values that might be useful.

Resolves #92

### CODE REVIEW

File: kubernetes/overlays/dev/local/environment-variables.env [hunk 1]
```diff
@@ -0,0 +1 @@
+LOCALHOST=172.25.61.231
```
Comment (YohannParis) [inline kubernetes/overlays/dev/local/environment-variables.env:1]: Are you sure you want to send this?

**Review by YohannParis — COMMENTED** [review]
This is only necessary for the `hmi-server` and the gateway?

File: kubernetes/overlays/dev/local/environment-variables.env (hunk 1 above)
Comment (tangopapatime) [reply kubernetes/overlays/dev/local/environment-variables.env]: :P

Comment (tangopapatime) [reply kubernetes/overlays/dev/local/environment-variables.env]: > This is only necessary for the `hmi-server` and the gateway?

Yeah.  The config is shared across all kustumizations and `$(LOCALHOST)` should be used going forward for any ip's that are supposed to be host ip.

**Review by bigglesandginger — APPROVED** [review]
Definitely works for Mac, which I get is not the point of the "Linux" section, but doesn't break that !  I will try to get my windows machine running with this as a test.

**Pull request merged**